### PR TITLE
feat: redesign electrs page

### PR DESCRIPTION
**Pull request by cstenglein**
Screenshot 2024-09-23 at 22-53-24 RaspiBlitz - myBlitz

### CODE REVIEW

**Comment by cstenglein** [comment]
FYI @escapedcat : i removed the `light` variant of the back button in the app page, it was hard to read that way and is consistent with the electrs page.

**Comment by cstenglein** [comment]
From @openoms :

QR Code needs a white border

File: src/pages/Apps/customApps/Electrs.tsx [hunk 1]
```diff
@@ -47,95 +57,80 @@ const Electrs: FC = () => {
...
+        </Button>
       </section>
-      <section className="bd-card mx-5 flex flex-col gap-4 px-5 lg:flex-row">
-        <article className="lg:w-1/2">
-          {!initialSyncDone && <p>{t("appInfo.electrs.initialSync")}</p>}
-          {initialSyncDone && (
+
+      <Card className="bd-card">
+        <CardHeader>
+          <h3 className="text-xl">
```
Comment (escapedcat) [inline src/pages/Apps/customApps/Electrs.tsx:78]: Wanna use `import { Headline } from "@/components/Headline";` for this?

This should work:
```
<Headline as="h3" size="xl">...
```

Comment (cstenglein) [reply src/pages/Apps/customApps/Electrs.tsx]: Done

**Comment by cstenglein** [comment]
Changed the Design a bit, now centered a bit more stuff.

Aligning it to the left like @rootzoll suggested wasn't looking that good IMO.

Screenshot 2024-09-24 at 16-51-29 RaspiBlitz - myBlitz
Screenshot 2024-09-24 at 16-52-06 RaspiBlitz - myBlitz
Screenshot 2024-09-24 at 16-52-21 RaspiBlitz - myBlitz

**Comment by cstenglein** [comment]
If the screen isn't wide enough, it clips the tor address.

Not sure how to handle it better. Copy works ofc, but you may not be able to see the whole address:

Screenshot 2024-09-24 at 16-56-11 RaspiBlitz - myBlitz
Screenshot 2024-09-24 at 16-56-20 RaspiBlitz - myBlitz

**Review by escapedcat — APPROVED** [review]

**Pull request merged**